src/content/docs/properties/about-properties/index.mdx

```diff
@@ -11,13 +11,13 @@ app_context:
     s: properties
 ---
 
-All business, user, and organization data stored in Kinde is made up of ‘properties’. Properties are single data points or fields, like email, first name, last name, phone number, address, etc.
+Properties are a way for you to add custom data to Kinde, and then use that data for custom claims and in tokens.
 
-Kinde comes with a standard set of properties that make up business, user, and organization profiles, but you can also [add custom properties](/properties/work-with-properties/manage-properties/) to capture any data you like.
+Kinde comes with a standard set of properties that make up business, user, application, and organization profiles, but you can also [add custom properties](/properties/work-with-properties/manage-properties/) to capture any data you like.
 
 For example, you might want to record a billing address separate to a delivery address, or capture screen names, job titles, or other customer information.
 
-Properties can also be used to create custom claims.
+Properties can also be used to create custom claims and to add data to M2M tokens.
```

Comment:
This was probably fine as it was before as it is also adding claims to M2M tokens, there isn't really a difference